christfellowshipchurch/web-app-v2
# Update feature feed

This PR removes the unused FeedFeaturesProvider and grabs all the features that need to be mapped from the mobile app. They are returning `null` so we can just build them out as the web 2.0 needs them. This should also fix the runtime error @dzwood was seeing about the FeatureProvider.

TO TEST:
Fire up the app and click around to make sure nothing is broken related to feature feeds.
